packages/amplication-client/src/VersionControl/PendingChange.scss

```diff
@@ -36,4 +36,16 @@
   &__version {
     font-weight: 500;
   }
+  &__tooltip_deleted:after {
+    border: 1px solid var(--negative);
+    background-color: var(--black10);
+    color: var(--black100);
+  }
+  &__deleted {
+    color: var(--black80);
+    cursor: pointer;
+  }
+  .tooltipped-ne:before {
+    border-top-color: var(--negative) !important;
+  }
```

Comment:
please add a screenshot of the new tooltip showing the styling you suggested. 
@alexbass86 will be able to review and comment or approve